frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.tsx

```diff
@@ -249,7 +648,64 @@ export const ProfilePage: React.FC<ProfilePageProps> = ({
                   })}
                 </Link>
               </div>
-              <p>Government status details</p>
+              <div
+                data-h2-bg-color="b(gray)"
+                data-h2-padding="b(all, m)"
+                data-h2-radius="b(s)"
+              >
+                {/* !!!!!!!!!!!!
+                TODO
+                ADD GOVERNMENT EMPLOYEE TYPE TO THIS FORM FUNCTION, THEN ADD EMPLOYEE TYPE LOGIC HERE
```

Comment:
This is the missing field you mentioned?  Could you add the issue # here?

Reply:
going to mention this component in #2329